friends/src/App.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './App.css';
+import axios from 'axios';
+import Friend from './components/friends'
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
This [article](https://medium.com/technical-credit/things-to-stop-start-doing-in-react-9887b9f6cea6) goes into this more, but now we can drop constructors and initialize state as a class property like below.
```suggestion
state = {
  friends: [],
  name: '',
  age: '',
  email: '',
  id: ''
}
```